analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -271,6 +271,23 @@ private fun String.isExternalDocumentReferenceId(): Boolean = startsWith(SpdxCon
  */
 private fun String.mapNotPresentToEmpty(): String = takeUnless { SpdxConstants.isNotPresent(it) }.orEmpty()
 
+/**
+ * Wrap any "present" SPDX value in a sorted set, or return an empty sorted set otherwise.
+ */
+private fun String?.wrapPresentInSortedSet(): SortedSet<String> {
+    if (this == null || SpdxConstants.isNotPresent(this)) return sortedSetOf()
+
+    val persons = withoutPrefix(SpdxConstants.PERSON)?.let { persons ->
+        // In case of a person, allow a comma-separated list of persons.
+        persons.split(',').mapTo(sortedSetOf()) { it.trim() }
+    }
+
+    // Do not split an organization like "Acme, Inc." by comma.
+    val organization = withoutPrefix(SpdxConstants.ORGANIZATION)?.let { sortedSetOf(it) }
+
+    return persons ?: organization ?: sortedSetOf()
```

Comment:
What is the reason to ignore the organization if persons were found?

Reply:
The [originator](https://spdx.github.io/spdx-spec/package-information/#761-description) is "Single line of text with *one* of the following keywords", so it cannot contain both "Person:" and "Organization:", and it's fine to only accept either one.

Reply:
Then you could early return inside the `let` blocks and omit the `val`s.